product-updates/knowledge-center/uploading-data.html

```diff
@@ -265,7 +276,11 @@ <h3 class="usa-process-list__heading">Submit your files</h3>
       <p>After uploading the files you want to submit, click the ‘Submit Data Files’ button. You should see a confirmation that your sections were submitted!</p>
       <a href="../img/data-files/4-success-message.png" data-lity data-lity-desc="Upload components showing success message above upload components reading 'Successfully submitted section(s): 1 on Wed Jun 15 2022'">
       <img src="../img/data-files/4-success-message.png" alt="Upload components showing success message above upload components reading 'Successfully submitted section(s): 1 on Wed Jun 15 2022'">
-        </a>
+      </a>
+      <p>Upon successful submission of your data files you will also receive an email confirmation from <em>no-reply@tanfdata.acf.hhs.gov<em> (example below).<p>
+      <a href="../img/getting-started/data-submitted.png" data-lity data-lity-desc="Email example of the successfully submitted data files by section">
+      <img src="../img/getting-started/data-submitted.png" alt="Email example of the successfully submitted data files by section">
```

Comment:
lets eventually get that correct email template added here @reitermb @lilybl1 
after 2.2

![knowctr](https://user-images.githubusercontent.com/63075587/200616081-f53fa4fb-07a8-48ab-a4e0-a5699b20d9e6.PNG)